### ISSUE TITLE
Sony SA-CT60 soundbar

### ISSUE THREAD

**Issue by Myrinero** [irplus-remote/irplus-codes.github.io#254]
Hello there,

I've lost the remote for the sounbar and cuz the lack of hardware buttons I can't even turn it on.
Try'd the app and it didn't work with soundbar codes for sony in it.

Thanks in advance.

**Comment by irplus-remote** [comment]
Hi, did you try the SA-CT Codes? Which Smartphone and which irplus Version are you using?

**Comment by Myrinero** [comment]
Yes, i try'd the SA-CT and HCT-Soundbar codes, but don't work. I'm using Alcatel idol 2 mini with android 4.3 and Huawei t1 7.0 tablet with android 4.4.2 and latest Irplus and Wave versions. Both devices don't have integrated ir, so i diy my self from your guide. Did two ir transmiters, to be sure that i did not mistake making it. Even diasemble the soundbar to have direct look at the transmiter.

On my old Galaxy s4 mini i had it working with Peel smart remote.

The model of the remote is RM-ANU159.

Thanks.

**Comment by irplus-remote** [comment]
Hi, please uninstall the regular irplus, and only use irplus WAVE (if you dont have integrated IR) to avoid issues.

Can you confirm that your DIY blaster is working correctly? Did you try to successfully control your TV? To confirm that the Voltage of your device is enough, try to see it lighting up with a camera.

I'm pretty sure that the SA-CT codes are correct.

S4-mini has integrated IR and that is a whole different story :)

**Comment by Myrinero** [comment]
Well, did leave only the Wave app, but as you said to try - cannot see it light with any of the devices, even try'd with the canon. Maybe the diods are dead or they are not the right one?
I'm going out now to take new ones

I don't use TV, so you know... don't have any. :D

**Comment by irplus-remote** [comment]
Canon? It should light with any button press (one-directional) it doesnt matter which device you set up basically. Could you post pictures (good ones) of your "construction". And if you have - post also the spec of the IR-LED used. Note they must have a low required voltage (about 1.1-1.2V). If you attach a battery to L+ GND / R+GND they should light up individually

**Comment by Myrinero** [comment]
https://drive.google.com/open?id=0B-CpBrJmtb-lZlE4bnNFdUhaWnM

Okay, with a battery one of the diods lights up, but with the tablet or phone - no. 
Don't know the specs of the ir's. Just bought ir diods from the shop (they have only one model at the time).

**Comment by irplus-remote** [comment]
Hm by the picture it's hard to tell if its wired up correctly, so if you connect to a battery (use 1.2V! not 1.5V!) GND+L one ligihts up, if you connect GND+R the other one lights up?

**Comment by Myrinero** [comment]
It's wired correctly. The same as the picture bellow:
https://wiki.samygo.tv/images/thumb/b/b2/Samsung_remote.jpg/450px-Samsung_remote.jpg

Think the problem it's that i'm using 3v diods. I cannot find anywhere 1.1-1.2v diods in local hardware shops, so i'm gonna purchase online to see is that the problem.

**Comment by irplus-remote** [comment]
With 3v LEDs it will never work. A smartphone can not provide this kind of power (not even near) via the audio out. you need about 1.1-1.2v 

Example with U_f = 1.2V

https://www.conrad.de/de/ir-emitter-940-nm-20-3-mm-rund-radial-bedrahtet-lite-on-lte-4206-1127307.html

**Comment by duczz** [comment]
Someone can tell me the IR code for the volume up down? my remote control is broken and i wanna controll it via IR now too
